src/CurlDownloader.php

```diff
@@ -27,7 +27,6 @@ class CurlDownloader
         'http' => [
             'method' => CURLOPT_CUSTOMREQUEST,
             'content' => CURLOPT_POSTFIELDS,
-            'proxy' => CURLOPT_PROXY,
```

Comment:
does it work if we only remove this line and don't add the other ones?
AFAIK, curl already reads from the env by default so there might be no need to do it in PHP.

Reply:
> does it work if we only remove this line and don't add the other ones?
> AFAIK, curl already reads from the env by default so there might be no need to do it in PHP.

yes that works too for me

Reply:
great, let's remove all added lines then :)

Reply:
Just tested. It works. Commit is coming (very soon).

Reply:
@nicolas-grekas Done :)